modules/management/kubectl-apply/kubectl/outputs.tf

```diff
@@ -0,0 +1,20 @@
+/**
+  * Copyright 2024 Google LLC
+  *
+  * Licensed under the Apache License, Version 2.0 (the "License");
+  * you may not use this file except in compliance with the License.
+  * You may obtain a copy of the License at
+  *
+  *      http://www.apache.org/licenses/LICENSE-2.0
+  *
+  * Unless required by applicable law or agreed to in writing, software
+  * distributed under the License is distributed on an "AS IS" BASIS,
+  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  * See the License for the specific language governing permissions and
+  * limitations under the License.
+  */
+
+output "documents_count" {
+  description = "The count of applying documents."
```

Comment:
nit: should we name this to `applied_documents_count` ? 

Also a quick question, say the caller intended to apply 10 manifests but the module was only able to apply 9 manifests, at the end of retry operations does it the module automatically revert the successfully applied 9 manifest ?